great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py

```diff
@@ -740,7 +740,7 @@ def _get_kl_divergence_chart(cls, partition_object, header=None):
                     .properties(width=chart_pixel_width, height=400, autosize="fit")
                 )
 
-                chart = bars.to_json()
+                chart = bars.to_dict()
```

Comment:
Why this change?

Reply:
This resulted in the Altair charts being serialized as a json string. It is preferable to pass this as a dictionary so the string doesn't need to be parsed later.